co-log/co-log.cabal

```
@@ -52,6 +52,7 @@ library
                      , stm >= 2.4 && < 2.6
                      , text ^>= 1.2.3
                      , time >= 1.8 && < 1.10
+                     , chronos >= 1.0.4
```

It's always better to add upper bounds as well for better compatibility.

```suggestion
                     , chronos ^>= 1.0.4
```